model/roles.go

```diff
@@ -134,22 +145,154 @@ func LoadRoleManifest(manifestFilePath string, releases []*Release) (*RoleManife
 	return &rolesManifest, nil
 }
 
+// SetGlobalConfig initializes the roleManifest's configuration of global entities
+// that it needs later to calculate the dev version:
+// the global properties, processed light opinions (to get the properties), and raw dark opinions
+func (m *RoleManifest) SetGlobalConfig(lightManifestPath, darkManifestPath string) error {
+	manifestConfig := manifestConfiguration{}
+	opinions, err := manifestConfig.loadOpinionsFromPath(lightManifestPath)
+	if err != nil {
+		return err
+	}
+	manifestConfig.lightOpinions = opinions
+	opinions, err = manifestConfig.loadOpinionsFromPath(darkManifestPath)
+	if err != nil {
+		return err
+	}
+	manifestConfig.darkOpinions = opinions
+	manifestConfig.globalDefaultValues = getDefaultValuesHash(m.Configuration.Templates)
+	m.globalConfig = &manifestConfig
```

Comment:
I believe it would be nicer to write the above as
```
m.globalConfig = &manifestConfiguration{
lightOpinions: lightOpinions,
darkOpinions: darkOpinions,
}
```
using two helper variables for the loaded data, and defers the allocation of the structure to after everything is loaded without error.